src/Nethermind/Nethermind.Consensus/Processing/BlockchainProcessor.cs

```diff
@@ -27,7 +27,7 @@
 
 namespace Nethermind.Consensus.Processing;
 
-public sealed class BlockchainProcessor : IBlockchainProcessor, IBlockProcessingQueue
+public sealed class BlockchainProcessor : IBlockchainProcessor, IBlockProcessingQueue, IAsyncDisposable
```

Comment:
We should probably change:

`IBlockchainProcessor : IDisposable` 

to:

`IBlockchainProcessor : IAsyncDisposable`

?

Reply:
Done